### PR TITLE
chore: update custom istio enricher (maven) quickstart to use fabric8…

### LINKED ISSUE #1505: [Quickstarts] : Update Custom Istio Enricher (Maven) quickstart to use Fabric8 Istio Extension
### Description

When we created this quickstart we were using Snowdrop Istio Extension for manipulating Istio objects. This extension was recently migrated to Fabric8: https://github.com/fabric8io/kubernetes-client/issues/3493

https://github.com/eclipse/jkube/blob/7e849a7dd04466ea10684e0c493c3e5dcd5db1ea/quickstarts/kit/custom-istio-enricher/pom.xml#L58-L62

This dependency should be available under fabric8 groupId on [maven central](https://search.maven.org/artifact/io.fabric8/istio-client/5.12.2/bundle):

```xml
<dependency>
  <groupId>io.fabric8</groupId>
  <artifactId>istio-client</artifactId>
  <version>5.12.2</version>
</dependency>
```

- [x] Change dependency Maven coordinates and version
- [x] Update code to use updated package names (`me.snowdrop.istio.*` -> `io.fabric8.istio.*`)
- [ ] Make sure quickstart is working correctly as per instructions provided in README

### PR DESCRIPTION
**Pull request by MMajd**
## Description
Fix #1505

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Review by rohanKanojia — APPROVED** [review]

File: quickstarts/kit/custom-istio-enricher/pom.xml [hunk 1]
```diff
@@ -56,9 +56,9 @@
         <scope>import</scope>
       </dependency>
       <dependency>
-        <groupId>me.snowdrop</groupId>
+        <groupId>io.fabric8</groupId>
         <artifactId>istio-client</artifactId>
-        <version>${istio-client.version}</version>
+        <version>5.12.2</version>
```
Comment (manusa) [inline quickstarts/kit/custom-istio-enricher/pom.xml:61]: Could you please replace the current variable value in L41 `<istio-client.version>1.7.5-Beta2</istio-client.version>` and still use it here?

**Review by manusa — CHANGES_REQUESTED** [review]
See comments

File: quickstarts/kit/custom-istio-enricher/pom.xml (hunk 1 above)
Comment (MMajd) [reply quickstarts/kit/custom-istio-enricher/pom.xml]: I have just committed this change now, also I very sorry about not  commenting on the issue or asking to be assigned on it before my PR

**Review by manusa — APPROVED** [review]

**Pull request merged**